### PR TITLE
Add calculate matrix to Routing API

### PR DESCRIPTION
**Pull request by jomarinb**
## Feature
Added a way to calculate matrices using a `herepy.RoutingApi` instance.

## Changes
- Replaced the package initializer from the tests directory with an empty `conftest.py` file in the project root. This prevents unexpected behaviors due to test modularity implications. [See the answer by hoefling](https://stackoverflow.com/questions/10253826/path-issue-with-pytest-importerror-no-module-named-yadayadayada).
- Added `matrix` method to `RoutingApi` class which wraps a call to the calculate matrix endpoint of Here's Routing API.
- Fixed some extra formatting typos found when parsing coordinates.

## Tests
- Added unit tests for the new `matrix` method.
- Manually tested against the remote Here API using personal credentials.

### CODE REVIEW

File: herepy/routing_api.py [hunk 1]
```diff
@@ -253,6 +259,39 @@ def truck_route(self,
...
+          departure (str):
+            time when travel is expected to start, e.g.: '2013-07-04T17:00:00+02'
+          modes (array):
+            array of RouteMode enums following [Type, TransportMode, TrafficMode, Feature].
+        Returns:
+          RoutingMatrixResponse
+        Raises:
+          HEREError: If an error is received from the server.
+        """
+        data = {
```
[the remark this answers is not in the document]
Comment (jomarinb) [reply herepy/routing_api.py]: Sorry. I didn't notice it. Done.

File: testdata/models/matrix.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            {
+                "startIndex": 0,
+                "destinationIndex": 0,
+                "summary": {
+                    "costFactor": 320
+                }
+            }
+        ]
+    }
+}
```
[the remark this answers is not in the document]
Comment (jomarinb) [reply testdata/models/matrix.json]: I completely agree. Done.

**Comment by jomarinb** [comment]
Hello @abdullahselek . Can I expect a new release after this gets merged?

File: testdata/models/routing_matrix_multiple_destinations.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            {
+                "startIndex": 0,
+                "destinationIndex": 1,
+                "summary": {
+                    "costFactor": 4694
+                }
+            }
+        ]
+    }
+}
```
Comment (abdullahselek) [inline testdata/models/routing_matrix_multiple_destinations.json]: Missing new line end of file.

File: testdata/models/routing_matrix_multiple_starts.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            {
+                "startIndex": 1,
+                "destinationIndex": 0,
+                "summary": {
+                    "costFactor": 1114
+                }
+            }
+        ]
+    }
+}
```
Comment (abdullahselek) [inline testdata/models/routing_matrix_multiple_starts.json]: Missing new line end of file.

**Comment by abdullahselek** [comment]
Of course after this PR I will release a new version.

**Comment by jomarinb** [comment]
I just address the last changes to reported.

Also, I added a parameter for the summary attributes to the `matrix` method.

Looking forward to your review.

**Comment by abdullahselek** [comment]
Looks good to me thanks for your contribution 👍

**Comment by abdullahselek** [comment]
@jomarinb `0.6.3.3` is available now.

**Pull request merged**